credentials/github.toml

```diff
@@ -0,0 +1,2 @@
+api_token = 
+organization = ubclaunchpad
```

Comment:
`organization` isn't really credentials - I would put it in the main config (the `ubclaunchpad` default should be fine though, yeah).

Reply:
Oh, and also, although I believe this works, I think it's "more proper" TOML to wrap the field (i.e. `ubclaunchpad`) in quotes.